sections/discussion/ourprocessreflections.tex

```diff
@@ -54,6 +54,20 @@ \subsubsection{Formal reviews}
     \label{fig:formal-reviews}
 \end{figure}
 
+\subsubsection{Prototypes}
+In the beginning of the semester we created multiple prototypes for the entire system so that everyone had the same understanding about how the design should be.
```

Comment:
```suggestion
In the beginning of the semester we created multiple prototypes for the entire system so that everyone had the same understanding of how the design should be.
```